query_server/query_operations.py

```diff
@@ -393,4 +399,3 @@ def discovery_programs():
     }
 
     return ret_val, 200
```

Comment:
```suggestion
    return ret_val, 200

```
Should keep newlines at the end of .py files, as per style guides (flake8 recommends it)